src/routes/db/productsModel.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const productsSchema = new Schema({
+  //_id: { type: mongoose.Types.ObjectId, required: true },
+  name: String,
+  model: String,
+  brand: String,
+  categoryId: String,
```

Comment:
categoryId should be `mongoose.Types.ObjectId` type

Reply:
swapped.